File: app/src/main/java/com/autobridge/android/BroadcastReceiver.java
package com.autobridge.android;

import android.content.Context;
import android.content.Intent;

public class BroadcastReceiver extends android.content.BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        context.startService(new Intent(context, Service.class));
    }
}

File: app/src/main/kotlin/com/autobridge/android/BroadcastReceiver.kt
package com.autobridge.android

import android.content.Context
import android.content.Intent

class BroadcastReceiver : android.content.BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        context.startService(Intent(context, Service::class.java))
    }
}
